pkg/apis/camel/v1/integration_types.go

```diff
@@ -184,12 +184,10 @@ const (
 	IntegrationConditionProbesAvailableReason string = "ProbesAvailable"
 	// IntegrationConditionErrorReason --
 	IntegrationConditionErrorReason string = "Error"
-	// IntegrationConditionCronJobCreatedReason --
-	IntegrationConditionCronJobCreatedReason string = "CronJobCreated"
-	// IntegrationConditionReplicaSetReadyReason --
-	IntegrationConditionReplicaSetReadyReason string = "ReplicaSetReady"
-	// IntegrationConditionReplicaSetNotReadyReason --
-	IntegrationConditionReplicaSetNotReadyReason string = "ReplicaSetNotReady"
+	// IntegrationConditionPodsReadyReason --
+	IntegrationConditionPodsReadyReason string = "PodsReady"
+	// IntegrationConditionPodsNotReadyReason --
+	IntegrationConditionPodsNotReadyReason string = "PodsNotReady"
```

Comment:
Should we take this as a potential minor breaking change ? i.e. for tools that are monitoring an integration, changing a condition could break them.

I don't have any objection but maybe we need to document it

Reply:
I agree, ideally we should have a "Breaking Changes" section in the release notes, that would document these kind of changes.